pywbem/cim_operations.py

```diff
@@ -1255,14 +1331,404 @@ class are to be included in the returned instances, as follows:
+
+            * If `False`, qualifiers are not included.
+            * If `True`, qualifiers are included if the WBEM server implements
+              support for this parameter.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default to be used. :term:`DSP0200`
+              defines that the server-implemented default is `False`.
+
+            This parameter has been deprecated in :term:`DSP0200`. Clients
+            cannot rely on it being implemented by WBEM servers.
+
+          IncludeClassOrigin (:class:`py:bool`):
+            Indicates that class origin information is to be included on each
+            property in the returned instances, as follows:
+
+            * If `False`, class origin information is not included.
+            * If `True`, class origin information is included.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default to be used. :term:`DSP0200`
+              defines that the server-implemented default is `False`.
+
+          PropertyList (:term:`py:iterable` of :term:`string`):
+            An iterable specifying the names of the properties to be
+            included in the returned instances, in any lexical case.
+
+            An empty iterable indicates to include no properties.
+
+            If `None`, all properties are included.
+
+          FilterQueryLanguage (:term:`string`):
+            Optional: a String defining the name of the query language
+            used for the Query argument. This argument must exist if the
+            filterQuery argument is provided.
+
+          QueryLanguage (:term:`string`):
+            Optional: a String defining the query that is to be sent
+            to the WBEM server.
+          #TODO define typing for this correctly
+          OperationTimeout (term: `Uint32`):
+            Optional: If provided it defines a timeout value to be
+            sent to the WBEM Server
+            * If `Uint32` value, this value is sent to the server as the
+              proposed timeout for the operation sequence. If the client
+              waits longer than this timeout in seconds, the server may
+              terminate the operation
+            * if `None`
+
+          ContinueOnError (:class:`py:bool`):
+            Indicates to the server to continue sending responses
+            after errorresponse has been sent.
+            * if `True` the server may continue sending responses after
+              sending an error response.
+            * If `False` the server must not continue sending responses after
+              and error response is sent and must close the enumeration
+              sequence after sending any error response.
+            * if `None` this parameter is not sent to the WBEM server and the
+              WBEM server is to close the enumeration sequence upon error.
```

Comment:
Instead, say: "this parameter is not passed to the WBEM server, and  causes the server-implemented default to be used. :term:`DSP0200`  defines that the server-implemented default is ......"